bootstrap/kubeadm/controllers/kubeadmconfig_controller_test.go

```diff
@@ -248,6 +248,10 @@ func TestKubeadmConfigReconciler_Reconcile_MigrateToSecret(t *testing.T) {
 	if string(secret.Data["value"]) != "test" {
 		t.Fatal("expected bootstrap data secret value to match")
 	}
+
+	if secret.Labels[clusterv1.ClusterLabelName] != "cluster" {
+		t.Fatalf("expected bootstrap data secret to have a cluster name label")
```

Comment:
We can use `t.Fatal` instead of `t.Fatalf`

Reply:
I actually wanted to add the formatting, thanks for reminding :)

Reply:
Done!